cmd/cosign/cli/options/sign.go

```diff
@@ -64,14 +64,8 @@ func AddSignOptions(cmd *cobra.Command, o *SignOptions) {
 	cmd.Flags().StringVar(&o.Attachment, "attachment", "",
 		"related image attachment to sign (sbom), default none")
 
-	AddAnnotationOptions(cmd, &o.AnnotationOptions)
+	o.AnnotationOptions.AddFlags(cmd)
 
 	cmd.Flags().BoolVar(&o.RegistryOpts.AllowInsecure, "allow-insecure-registry", false,
 		"whether to allow insecure connections to registries. Don't use this for anything but testing")
-
-	AddRekorOptions(cmd, &o.Rektor)
```

Comment:
these get lost?

Reply:
Must have been a bad merge.  I'd moved them to the top.  I'll add them to my push fixing the style thing.

Reply:
there now, 👍